cms/static/js/views/move_xblock_list.js

```diff
@@ -26,14 +26,17 @@ function($, Backbone, _, gettext, HtmlUtils, StringUtils, XBlockUtils, MoveXBloc
             course: 'section',
             section: 'subsection',
             subsection: 'unit',
-            unit: 'component'
+            unit: 'component',
+            split_test: 'group',
+            group: 'component'
```

Comment:
@mushtaqak @muzaffaryousaf We need to discuss about sending this map from server instead of hardcoding here? I am worried things might break at some point due to this? What will happen if a new parent type is added to studio? Should we have a test that will fail if we encountered an XBlock with category other than the above parent/child categories?